frontend/src/services/storage.js

```diff
@@ -0,0 +1,49 @@
+const tokenKeyName = 'auth.access_token';
+class Storage {
+    constructor(type = 'localStorage') {
+        this.store = window[type];
+    }
+
+    get(key) {
+        return this.store.getItem(key);
+    }
+
+    set(key, value) {
+        return this.store.setItem(key, value);
+    }
+
+    removeItem(key) {
+        this.store.removeItem(key);
+    }
+
+    getToken() {
+        return this.get(tokenKeyName);
+    }
+
+    setToken(token) {
+        return this.set(tokenKeyName, token);
+    }
+
+    hasToken() {
+        return !!this.get(tokenKeyName);
+    }
+
+    removeToken() {
+        return this.removeItem(tokenKeyName);
+    }
+}
+export const sessionStorage = new Storage('sessionStorage');
+export const localStorage = new Storage();
+
+isLocalStorageAccessible = () => try { 
```

Comment:
you weren't supposed to copy my code)) but anyway, please, add const to function and use curly braces when you define function in this case, because it won't work. Get rid of exports of sessionStorage and localStorage variables.  And test it, please